build.gradle

```diff
@@ -0,0 +1,78 @@
+plugins {
+	id 'java'
+	id 'org.springframework.boot' version '2.3.3.RELEASE'
+	id 'io.spring.dependency-management' version '1.0.10.RELEASE'
+    id 'org.jetbrains.kotlin.jvm' version '1.3.72'
+}
+
+group = 'com.cag.cagbackendapi'
+version = '0.0.1-SNAPSHOT'
+sourceCompatibility = '11'
+
+repositories {
+	mavenCentral()
+}
+
+dependencies {
```

Comment:
How do you know what the dependencies are? Do you have to write them line by line?

Reply:
Dependencies are one of the most important parts of the build.gradle because this tells the project what libraries to pull from https://mvnrepository.com/ where most java projects pull their libraries from.